cdn/snippets.py

```diff
@@ -23,199 +23,195 @@
     expiration_timestamp = int((expiration_time - epoch).total_seconds())
     decoded_key = base64.urlsafe_b64decode(base64_key)
 
-    policy_pattern = u'URLPrefix={encoded_url_prefix}&Expires={expires}&KeyName={key_name}'
-    policy = policy_pattern.format(
-            encoded_url_prefix=encoded_url_prefix,
-            expires=expiration_timestamp,
-            key_name=key_name)
+    policy = f"URLPrefix={encoded_url_prefix}&Expires={expiration_timestamp}&KeyName={key_name}"
+
+    digest = hmac.new(decoded_key, policy.encode("utf-8"), hashlib.sha1).digest()
+    signature = base64.urlsafe_b64encode(digest).decode("utf-8")
 
-    digest = hmac.new(
-            decoded_key, policy.encode('utf-8'), hashlib.sha1).digest()
-    signature = base64.urlsafe_b64encode(digest).decode('utf-8')
+    return f"{stripped_url}{'&' if query_params else '?'}{policy}&Signature={signature}"
 
-    signed_url = u'{url}{separator}{policy}&Signature={signature}'.format(
-            url=stripped_url,
-            separator='&' if query_params else '?',
-            policy=policy,
-            signature=signature)
 
-    print(signed_url)
 # [END cloudcdn_sign_url_prefix]
 
 
 # [START cloudcdn_sign_cookie]
-def sign_cookie(url_prefix, key_name, base64_key, expiration_time):
+def sign_cookie(
+    url_prefix: str,
+    key_name: str,
+    base64_key: str,
+    expiration_time: datetime,
+) -> str:
     """Gets the Signed cookie value for the specified URL prefix and configuration.
 
     Args:
-        url_prefix: URL prefix to sign as a string.
-        key_name: name of the signing key as a string.
+        url_prefix: URL prefix to sign.
+        key_name: name of the signing key.
         base64_key: signing key as a base64 encoded string.
-        expiration_time: expiration time as a UTC datetime object.
+        expiration_time: expiration time.
 
     Returns:
         Returns the Cloud-CDN-Cookie value based on the specified configuration.
     """
     encoded_url_prefix = base64.urlsafe_b64encode(
-            url_prefix.strip().encode('utf-8')).decode('utf-8')
-    epoch = datetime.datetime.utcfromtimestamp(0)
+        url_prefix.strip().encode("utf-8")
+    ).decode("utf-8")
+    epoch = datetime.utcfromtimestamp(0)
     expiration_timestamp = int((expiration_time - epoch).total_seconds())
     decoded_key = base64.urlsafe_b64decode(base64_key)
 
-    policy_pattern = u'URLPrefix={encoded_url_prefix}:Expires={expires}:KeyName={key_name}'
-    policy = policy_pattern.format(
-            encoded_url_prefix=encoded_url_prefix,
-            expires=expiration_timestamp,
-            key_name=key_name)
+    policy = f"URLPrefix={encoded_url_prefix}:Expires={expiration_timestamp}:KeyName={key_name}"
+
+    digest = hmac.new(decoded_key, policy.encode("utf-8"), hashlib.sha1).digest()
+    signature = base64.urlsafe_b64encode(digest).decode("utf-8")
+
+    signed_policy = f"Cloud-CDN-Cookie={policy}:Signature={signature}"
+
+    return signed_policy
 
```

Comment:
ditto: please remove empty lines before the closure of the region tag if `black` formatter doesn't enforce them.